client/jetpack-cloud/sections/site-landing.jsx

```diff
@@ -16,24 +14,11 @@ export default function SiteLanding( { primarySiteSlug, isPrimarySiteJetpackSite
 	const hasFetched = useSelector( hasFetchedPartner );
 	const isFetching = useSelector( isFetchingPartner );
 	const dashboardRedirectLink = useSelector( jetpackDashboardRedirectLink );
-	const isAgency = useSelector( isAgencyUser );
-	const isAgencyEnabled = config.isEnabled( 'jetpack/agency-dashboard' );
 	useEffect( () => {
 		if ( hasFetched ) {
-			if ( isAgency && isAgencyEnabled ) {
-				page.redirect( dashboardRedirectLink );
-				return;
-			}
 			isPrimarySiteJetpackSite ? page( `/landing/${ primarySiteSlug }` ) : page( `/landing` );
 		}
-	}, [
-		primarySiteSlug,
-		hasFetched,
-		isPrimarySiteJetpackSite,
-		isAgency,
-		isAgencyEnabled,
-		dashboardRedirectLink,
-	] );
+	}, [ primarySiteSlug, hasFetched, isPrimarySiteJetpackSite, dashboardRedirectLink ] );
```

Comment:
I think now that we have a key `has_jetpack_partner_access`, we no longer need this component itself. We could move this logic to the controller. 

```isPrimarySiteJetpackSite ? page( `/landing/${ primarySiteSlug }` ) : page( `/landing` )``` logic can be moved to the controller as rest of the things here are no more required.

Reply:
I moved this logic to the controller along with the agency dashboard redirect logic.